src/app/view/components/_common/site-selector/site-selector.component.ts

```diff
@@ -157,4 +169,14 @@ export class SiteSelectorComponent implements OnInit, OnChanges {
             });
         }
     }
+
+    private isCurrentSiteOnListOfAvailableSites(sites: any[], currentSite: Site): boolean {
+        return sites.find((site) => {
+            return (site.identifier === currentSite.identifier && !site.archived);
+        });
+    }
+
+    private getDefaultSite(sites: any[]): Site {
+        return sites.filter((site) => site.default === true)[0];
```

Comment:
You don't need to compare ```=== true```, you can just do

```
return sites.filter((site) => site.default)[0];
```

Reply:
done